src/components/utils/__tests__/TimeUtils.test.js

```diff
@@ -1,89 +1,69 @@
-import moment from 'moment'
-
 import TimeUtils from '../TimeUtils'
 
+const { unix, seconds, minutes, hours, days } = TimeUtils
+
 // setup mocked date generation
 let now = null
+let nowYear = null
 beforeAll(() => {
-    // whenever moment() is invoked 2017-03-22 will be taken as date
-    Date.now = jest.fn(() => new Date(Date.UTC(2017, 2, 22)).valueOf())
-    now = moment()
+    // whenever Date.now() is invoked 2017-03-22 will be taken as date
+    // This was required to set the date within TimeUtils.js
+    global.Date.now = jest.fn(() => new Date(Date.UTC(2017, 2, 22)).valueOf())
+
+    now = Date.now()
+    nowYear = new Date(now).getFullYear()
+})
+
+afterAll(() => {
+    global.Date.now.mockRestore()
 })
 
 describe('Time Utils', () => {
```

Comment:
Could you add some no brainer unit tests for the new functions (unix, seconds, minutes, hours, days, getMinutes, getHours, getDays)?